### ISSUE TITLE
Calling an instance method on a object which class name ends with Globals is impossible

### ISSUE THREAD

**Issue by lgrignon** [cincheo/jsweet#132]
``` java
class SomethingGlobals {
  void mymethod();
}
SomethingGlobals myGlobals = ...;
myGlobals.mymethod();

```

At transpilation, myGlobals identifier is removed while calling mymethod. Since it is an instance method, we probably shouldn't remove this call.

**Comment by foopq** [comment]
I've been running into this as well:

`public class BlahGlobals {
    int i;
    int f() { return 5; }
}
...
BlahGlobals bg = new BlahGlobals();
int ii = bg.i;
int ff = bg.f();
`

transpiles to:

`let bg : BlahGlobals = ;
let ii : number = bg.i;
let ff : number = f();`

The language guide talks about Globals classes, but it seems to suggest only classes named exactly 'Globals'.

**Comment by lgrignon** [comment]
Hello @foopq , Globals class must only contain static members statically accessed because the class itself is erased at runtime.
The definition of what is a Globals class in the code is any class ending with "Globals", it works for static access but has undocumented behavior for instance access.
I let this issue opened to take a look when possible, in the mean time I suggest you just rename the class even if your case seems a test class :)
Could you please specify the version of JSweet you are using?

Feel free to submit a PR on the master (v2) branch fixing this problem.

Thanks for reporting.

**Comment by foopq** [comment]
Thanks for the clarification, I'm using "JSweet transpiler version 1.2.0 (build date: 2016-12-16 11:22:33)".

**Comment by renaudpawlak** [comment]
This is fixed in 2.0.0-SNAPSHOT (master).
